db/dcrpg/queries.go

```diff
@@ -1585,12 +1616,11 @@ func scanAddressQueryRows(rows *sql.Rows, queryType int) (ids []uint64, addressR
 	for rows.Next() {
 		var id uint64
 		var addr dbtypes.AddressRow
-		var txHash sql.NullString
-		var blockTime dbtypes.TimeDef
+		var matchingTxHash sql.NullString
```

Comment:
`txHash` is never nul or empty.  `matchingTxHash` is often empty, but I'm not even sure it is allowed to be null.  This is just to be safe.